packages/web-components/src/components/va-crisis-line-modal/va-crisis-line-modal.tsx

```diff
@@ -125,7 +125,7 @@ export class VACrisisLineModal {
               </a>
             </li>
             <li>
-              <va-icon icon="deaf" class="va-clm__icon" size={3}></va-icon>
+              <va-icon icon="tty" class="va-clm__icon" size={3}></va-icon>
```

Comment:
The new tty icon was missing so I'm adding it now:

![Screenshot 2024-06-11 at 10 23 38 AM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/29fddb34-2f04-4d3c-9698-9966e5de3f26)